types/office-js-preview/index.d.ts

```diff
@@ -17514,17 +17514,30 @@ declare namespace Office {
          *
          * - New Mac UI, Android: No limit
          *
-         * **Important**: In Outlook on the web, if a user created a new message by activating a contact's email address link from their contact or
-         * profile card, your add-in's `getAsync` call currently won't return a value in the `displayName` property of the
-         * associated `EmailAddressDetails` object. For more details, refer to the
-         * {@link https://github.com/OfficeDev/office-js/issues/2201 | related GitHub issue}.
-         *
          * @remarks
          * [Api set: Mailbox 1.1]
          *
          * **{@link https://docs.microsoft.com/office/dev/add-ins/outlook/understanding-outlook-add-in-permissions | Minimum permission level}**: `ReadItem`
          *
          * **{@link https://docs.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Compose
+         * 
+         * **Important**: In Outlook on the web and Windows, if a user creates a new message by activating a contact's email address link from their contact
```

Comment:
here and other instances
```suggestion
         * **Important**: In Outlook on the web and on Windows, if a user creates a new message by activating a contact's email address link from their contact
```